tests/system_health/test_system_health.py

```diff
@@ -49,6 +50,13 @@
 EXPECT_PSU_INVALID_VOLTAGE = '{} voltage is out of range'
 
 
+@pytest.fixture(autouse=True, scope="module")
+def check_image_version(duthost):
+    """Skip the test for unsupported images."""
+    pytest_require("201911" not in duthost.os_version, "Test not supported for 201911 images. Skipping the test")
```

Comment:
If the 2020 branch has newer kernel version, then it is more robust to skip this test on 201911 branch by kernel_version.

Example: 
https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/cli/test_show_platform.py#L118

Reply:
Great suggestion. Updated. Thanks.